src/gsIeti/gsPrimalSystem.h

```diff
@@ -0,0 +1,205 @@
+/** @file gsPrimalSystem.h
+
+    @brief This class represents the primal system and allows to incorporate the primal constraints
+
+    This file is part of the G+Smo library.
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Author(s): S. Takacs
+*/
+
+#pragma once
+
+#include <gsSolver/gsLinearOperator.h>
+#include <gsSolver/gsMatrixOp.h>
+#include <gsMatrix/gsVector.h>
```

Comment:
probably some inclusions can be removed?

Reply:
Removed unused